test-suite/coq-makefile/native4/run.sh

```diff
@@ -31,6 +36,3 @@ sort > desired <<EOT
 ./test/.coq-native/Ntest_test.cmxs
```

Comment:
@ppedrot IIUC this tests `-native-compiler no` in _CoqProject, so why do we expect coq-native files?